lib/src/aggregate_sample.dart

```diff
@@ -138,7 +138,7 @@ extension AggregateSample<T> on Stream<T> {
         // Handle opt-out nulls
         cancels.removeWhere((Object? f) => f == null);
         if (cancels.isEmpty) return null;
-        return Future.wait(cancels).then((_) => null);
+        return cancels.wait.then((_) => null);
```

Comment:
Could all these `(_) => null` use the same top-level helper function, `void _ignore([void _, void _]) {}`?
That should ensure we only have one function instance, instead of one per call point.